src/licensedcode/data/licenses/acm-sla.LICENSE

```diff
@@ -0,0 +1,95 @@
+---
+key: acm-sla
+short_name: ACM SLA
+name: ACM Software License Agreement
+category: Proprietary Free
+owner: ACM
+homepage_url: https://www.acm.org/publications/policies/software-copyright-notice
+spdx_license_key: LicenseRef-scancode-acm-sla
+other_urls:
+    - https://github.com/search?q="ACM+Software+License+Agreement"&type=code
+ignorable_copyrights:
+    - copyrighted by ACM
+ignorable_holders:
+    - ACM
+ignorable_authors:
+    - the original work and others
+ignorable_emails:
+    - permissions@acm.org
+---
+
```

Comment:
nit: there is no reason for an empty line there IMHO

Reply:
This is there in all licenses btw, we could fix in all licenses then but would be too much churn.

Reply:
OK, let's not change this yet... but this is an extra 30KB if we have 30000 empty lines ;)

Reply:
yup, we should definitely consider this, whatever space we can save matter.